### PR TITLE
Add planet coords function

### PR DESCRIPTION
**Pull request by TestSubjector**
Note - This function is a bit different than the one provided in IDL AstroLib.

* The option to compute positions using the full JPL ephemeris is not included, for now.
* Instead of planet name, an integer `num` denoting the planet number is taken as input (similar to as in helio - #27), which is made compulsory rather than optional for simpler code.
* The date input can either be in `DateTime` or `Real`(*julian date*, multiple dispatch [attempt number 2](https://github.com/JuliaAstro/AstroLib.jl/pull/13#issuecomment-297118125)) which also removes the need of optional keyword `JD`

### CODE REVIEW

File: src/planet_coords.jl [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+```
+
+### Notes ###
+
+Code of this function is based on IDL Astronomy User's Library.
+"""
+planet_coords(date::Real, num::Integer) = _planet_coords(float(date), num)
+
+function planet_coords(date::AbstractVector{R},
+                       num::AbstractVector{<:Real}) where {R<:Real}
```
Comment (giordano) [inline src/planet_coords.jl]: `num` should be `AbstractVector{<:Integer}`, right?

File: src/planet_coords.jl [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    typedate = float(R)
+    ra_out  = similar(date, typedate)
+    dec_out = similar(date, typedate)
+    for i in eachindex(date)
+        ra_out[i], dec_out[i] = planet_coords(date[i], num[i])
+    end
+    return ra_out, dec_out
+end
+
+planet_coords(date::DateTime, num::Integer) = planet_coords(juldate(date), num)
```
Comment (giordano) [inline src/planet_coords.jl]: Are you sure `juldate` should be used here?  `juldate` gives the *reduced* Julian date (Julian date - 2400000), while `jdcnv` gives the plain Julian date.  `helio` expects the plain Julian date

File: src/planet_coords.jl [hunk 3]
```diff
@@ -54,7 +54,7 @@ Find the RA, Dec of Venus on 1992 Dec 20
 
 ```jldoctest
 julia> adstring(planet_coords(DateTime(1992,12,20),2))
-" 02 12 22.7  +12 44 45"
+" 21 00 15.3  -19 09 09"
```
Comment (giordano) [inline src/planet_coords.jl]: Is this change due to https://github.com/JuliaAstro/AstroLib.jl/pull/30/commits/f1ce1be493a934d0aa490fcccd827bbf83e0f583?

Comment (TestSubjector) [inline src/planet_coords.jl]: Nope. It was an incorrect example from the before that.

Comment (giordano) [inline src/planet_coords.jl]: Why is there this difference with IDL AstroLib?
```
GDL> planet_coords, [1992,12,20], ra, dec, planet = "Venus"
GDL> print, adstring(ra, dec)
 21 05 02.8  -18 51 41
```

Comment (giordano) [inline src/planet_coords.jl]: Ok, this has probably to do with the inaccurate method to solve the Kepler's equation in `helio.pro`: I compared the mean anomalies in `helio.pro` and `helio.jl` for a few different inputs and they are exactly equal.

**Pull request merged**